ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2847,6 +2868,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-remote-worker
+- as: vsphere-ipi-sdn-migration-ovn-f14
+  cron: 36 19 13,28 * *
+  steps:
+    cluster_profile: vsphere-2
+    env:
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
+      UPGRADE_CHECK_RUN_TAGS: '@4.15 and @vsphere-ipi and @upgrade-check and @network-ovnkubernetes
```

Comment:
I think we do not need the TAG setting now.

Reply:
yes,  I removed them. thanks.

Reply:
cc @liangxia to confirm if I am wrong.

Reply:
Ack